protocol/v2/ssv/runner/runner.go

```diff
@@ -257,19 +258,29 @@ func (b *BaseRunner) baseConsensusMsgProcessing(ctx context.Context, logger *zap
 		prevDecided, _ = b.State.RunningInstance.IsDecided()
 	}
 	if prevDecided {
-		return true, nil, errors.New("not processing consensus message since consensus has already finished")
+		return true, nil, spectypes.NewError(spectypes.SkipConsensusMessageAsConsensusHasFinishedErrorCode, "not processing consensus message since consensus has already finished")
```

Comment:
Perhaps, we could go on with improving this and eventually use something shorter, not in this PR, though